### DISCUSSION TITLE
HashiCorp switched the license for Terraform from the MPL to the Business Source License (v1.1) (the “BUSL”), a non-open source license.

### DISCUSSION THREAD

**Discussion by hlange4** [opentelekomcloud/terraform-provider-opentelekomcloud#2358] [Ideas]
Should we as Terraform-Provider consider to also integrate us into https://opentofu.org/ ?
Previously named OpenTF, OpenTofu is a fork of Terraform that is open-source, community-driven, and managed by the Linux Foundation.

**Comment by anton-sidelnikov** [comment]
Published in opentofu registry
